javascript/javascript3/week1/gif.js

```diff
@@ -0,0 +1,42 @@
+const searchWord = document.getElementById("search-word")
+const btnSearch = document.getElementById("search-btn")
+
+const div = document.getElementById("giphy-content");
+btnSearch.addEventListener("click", getGiph);
+function getGiph() { 
+    const giphSearch = searchWord.value;
+    const countNumber = document.getElementById("count-number").value;    
+    if (searchWord === "")
+    div.innerHTML = 'pl enter valid search'; 
+      else
+      {let url;
+      if (countNumber > 0)
+        url = `http://api.giphy.com/v1/gifs/search?q=${giphSearch}&api_key=EmYRO1VcBqwrBZ9EQrRORj7YAzAEy68V&limit=${countNumber}`;
+      else
+        url = `http://api.giphy.com/v1/gifs/search?q=${giphSearch}&api_key=EmYRO1VcBqwrBZ9EQrRORj7YAzAEy68V`;
+      console.log(url);
+      console.log(countNumber);
+      getGiphy(url);}
+  }
```

Comment:
Are you using prettier code formatter to format your code in VS? If not, you please download that extension and try. 
I know that we are allowed to use if/else block without curly brackets, but in my point of view, it would be better for others to understand your code easily if you use curly brackets.